roles/postgres_tuning/tasks/main.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: Add postgres tuning configuration
+  copy:
+    content: |
+      shared_buffers = {{ ( ansible_memtotal_mb / 5 ) | int }}MB
+      effective_cache_size = {{ ( ansible_memtotal_mb / 2 ) | int }}MB
```

Comment:
Does this mean that Postgres may use half of the total memory of the server?

Reply:
My understanding of `effective_cache_size` is that it doesn't change how much memory is actually used, it just gives an indication to Postgres as to how much memory the system has, which Postgres uses when making estimates about how to plan queries.